src/ceph-volume/ceph_volume/tests/functional/lvm/tox.ini

```diff
@@ -39,7 +39,7 @@ changedir=
 commands=
   git clone -b {env:CEPH_ANSIBLE_BRANCH:master} --single-branch https://github.com/ceph/ceph-ansible.git {envdir}/tmp/ceph-ansible
 
-  vagrant up --no-provision {posargs:--provider=virtualbox}
+  vagrant up {env:VAGRANT_EXTRA_FLAGS:""} --no-provision {posargs:--provider=virtualbox}
```

Comment:
Are you sure all these vagrant commands take the same flags? Whatever is used for ``VAGRANT_EXTRA_FLAGS`` will need to work with ``up``, ``reload`` and ``destroy``.

Reply:
My idea was: "vagrant extra flags == single place that can be used everywhere". In this case "--debug" can be used all over (a.k.a. 'global flag'). We can add ad-hoc ones as well, would you prefer separate, per-sub-command env vars? Might be a little bit more verbose to add them, but could give you far better granularity, which is not a bad idea

Reply:
Maybe we should just change the name to ``VAGRANT_GLOBAL_EXTRA_FLAGS`` here and we can add subcommand specific ones if we find need for it.